### DISCUSSION TITLE
Does Thymeleaf support LocalDate and java.time.*?

### DISCUSSION THREAD

**Discussion by ge0ffrey** [thymeleaf/thymeleaf#907] [Q&A]
I searched for "LocalDate" and "date" in the Thymeleaf 3 docs, but could find no entry on it or `java.time.*` in general.

Does thymeleaf support displaying a `LocalDate` property (or `LocalTime` or `LocalDateTime`) in a template with minimal code? Without converting to java.util.Date?

**Comment by tikorth** [comment]
replace dates.format with temporals.format will work 

Em 18/08/2022 08:44, Geoffrey De Smet escreveu:

> I searched for "LocalDate" and "date" in the Thymeleaf 3 docs, but could find no entry on it or java.time.* in general. 
> 
> Does thymeleaf support displaying a LocalDate property (or LocalTime or LocalDateTime) in a template with minimal code? Without converting to java.util.Date? 
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe [2].
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
 

Links:
------
[1] https://github.com/thymeleaf/thymeleaf/discussions/907
[2]
